cookbook/catalog_structure/how_to_add_a_custom_unit_of_measure.rst

```diff
@@ -0,0 +1,86 @@
+How to add a custom unit of measure
+===================================
+
+If Akeneo PIM didn´t provide all the measures you need, you can easily add a custom unit of measure on an existing family.
+
+The custom measures are configured by a ``Resources/config/measure.yml`` file.
+This Yaml file could be in every custom bundle registered by the AppKernel.php and will be processed for the Akeneo MeasureBundle.
+
+In this example we will add the MICROMETER unit to the length measure family in a own MyMeasure Bundle.
+
+Create our Bundle
+-----------------
+
+Create a new Symfony bundle:
+
+.. code-block:: php
+	
+	<?php
+
+	namespace Acme\Bundle\MyMeasureBundle;
+
+	use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+	class AcmeMyMeasureBundle extends Bundle
+	{
+	}
+	
+Register the bundle in AppKernel:
+
+.. code-block:: php
+
+    public function registerProjectBundles()
+    {
+        // ...
+            new Acme\Bundle\MyMeasureBundle\AcmeMyMeasureBundle(),
+        // ...
+    }
+
+Configure our unit of measure
+-----------------------------
+	
+Create a file ``Resources/config/measure.yml`` in our Bundle to configure the unit of measure:
+
+.. code-block:: yaml
+	
+    measures_config:
+    Length:
+        standard: METER
```

Comment:
I don't think the `standard` should be in this file. It is already in the PIM config file and it could be very bad if it was modified.